### ISSUE TITLE
Empty npm view response causes failure

### ISSUE THREAD

**Issue by Adelrisk** [vscode-contrib/vscode-versionlens#173]
**tl;dr:**
A single npm dependency without metadata (as seen with `npm view`) breaks the extension for an entire file.

**In full**
I cannot view the dependency information for some package.json files. I used the developer tools to view addition error information and turned up this error statement:

> Error: NPM: parseNpmArguments [object Object]

reported here https://github.com/vscode-contrib/vscode-versionlens/blob/17b5ebc7e3a38218b4187701964a1b5ed3b67769/src/providers/npm/npmPackageParser.js#L71

``` javascript
{code: "NPM_VIEW_EMPTY_RESPONSE", message: "NPM view did not return any tags for @mycompany/our_package"}
```

This package @mycompany/our_package exists on a private npm registry with broken/incomplete metadata, as `npm view @mycompany/our_package` returns nothing. However, `npm install @mycompany/our_package` results in the correct installation of the latest version.

I know the real cause of the error lies with this package on our server, but it would be great if this extension was just a bit robuster.

**Comment by pflannery** [comment]
@Adelrisk Yeah this extension uses the npm view api to read back version information so error is in the npm api. The error could be handled better for usre. I dont have a private npm registry at hand at the moment so can't really fix this yet.

**Comment by pflannery** [comment]
@Adelrisk  this should be fixed in 0.26.2.

**Comment by Adelrisk** [comment]
Thanks for the work! I really appreciate the overview and context this plugin enables.
